crates/components/wick-http-client/src/component.rs

```diff
@@ -357,6 +363,22 @@ fn output_task(
 ) -> BoxFuture<'static, ()> {
   let task = async move {
     match codec {
+      Codec::EventStream => {
```

Comment:
Handling SSE responses is more of a special handling of a response type, vs a codec. The codec would describe how to handle the data in an event.

Since the `content-type: text/event-stream` is a standard, should this be handled automatically? The data would be processed as whatever the codec dictates before being sent to the output?